README.adoc

```diff
@@ -574,6 +574,30 @@ class SomethingTest < Minitest::Test
 end
 ----
 
+=== Skipping [[skipping]]
+
+Prefer `skip` over `return` for skipping test cases.
+
+[source,ruby]
+----
+
+# bad
+
+def test_something
+  return unless condition?
+
+  # do testing
```

Comment:
It would be preferable to show assertion method rather than source code comment.

```suggestion
  assert_equal(42, something)
```